workshop/05_Refactoring/03_Copy_Clone.php

```diff
@@ -14,7 +14,4 @@
 //    Do this from the Project Tool Window and do not touch the mouse.
 //    HINT: Alt+1/Command+1, arrow keys, F5
 
-// 2. Clone the current file and name it "03_Copy_Clone_(cloned).php".
-//    Use the Shift+F5 keyboard shortcut from within the editor.
-
-// 2. Copy the current file to the Code directory. Drag/drop it using Ctrl key pressed.
+// 2. Copy the current file to the Code directory. Drag/drop it using Ctrl key pressed.
```

Comment:
This is a valid scenario. It should be just:
// 3. Copy the current file to the Code directory. Drag/drop it using Ctrl key pressed.
instead of
// 2. Copy the current file to the Code directory. Drag/drop it using Ctrl key pressed.